routers/users.py

```
@@ -0,0 +1,31 @@
+from pydantic import EmailStr
+
+from common.config import settings
+from common.dependencies import get_auth_service
+from schemas.user import SignupRequest, UserResponse
+from common.common import CommonResponse
+from services.user_service import UserService
+router= APIRouter(prefix="/users",tags=["users"])
+@router.post(path= "/",response_model= CommonResponse[UserResponse],status_code=status.HTTP_201_CREATED)
+async def signup(
+    user_service: Annotated[UserService, Depends(get_auth_service)],
+    email: EmailStr = Form(...), #...은 requirement(필수), EmailStr (@ 형식으로 들어온 데이터)
```

Ruff (0.14.11)

Do not perform function call `Form` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)